mix.exs

```diff
@@ -197,7 +197,7 @@ defmodule AshPostgres.MixProject do
         [path: "../ash", override: true]
 
       "main" ->
-        [git: "https://github.com/ash-project/ash.git"]
+        [git: "https://github.com/ash-project/ash.git", override: true]
```

Comment:
If I don't add `override`, then I'm not able to use `main` option for `ASH_VERSION` variable.

Reply:
Interesting, I haven't had that problem but adding that should fine.

Reply:
Without `override: true`, I was getting a following error:

```
Dependencies have diverged:
* ash (https://github.com/ash-project/ash.git)
  the dependency ash in mix.exs is overriding a child dependency:

  > In mix.exs:
    {:ash, [env: :prod, git: "https://github.com/ash-project/ash.git"]}

  > In deps/ash_sql/mix.exs:
    {:ash, "~> 3.0 and >= 3.4.60", [env: :prod, hex: "ash", repo: "hexpm"]}

  Ensure they match or specify one of the above in your deps and set "override: true"
** (Mix) Can't continue due to errors on dependencies
```